### DISCUSSION TITLE
[Question][Hw3] Is there any API functions to change type into llvalue??

### DISCUSSION THREAD

**Discussion by Quasar0311** [prosyslab-classroom/cs348-information-security#549] [Q&A]
When I use build_call function, I have to put argument as a llvalue type, but while getting position of instruction, I have string value of them.

So, is there any API to translate normal string or integer value into llvalue?

**Comment by mickowale** [comment]
Take a look at 'operations on scalar constants' section on the ocaml Llvm API documatation site.

**Comment by Quasar0311** [comment]
Got it! Thanks!
